src/test/java/eu/merloteducation/contractorchestrator/ContractStorageServiceTest.java

```diff
@@ -1135,6 +1227,8 @@ void transitionDataDeliveryContractPurge() throws IOException {
         representedOrgaIds.add(consumer);
         representedOrgaIds.add(provider);
         DataDeliveryContractTemplate template = new DataDeliveryContractTemplate(dataDeliveryContract, false);
+        template.setServiceContractProvisioning(new DataDeliveryProvisioning()); // reset provisioning
```

Comment:
why is this saved after each step?

Reply:
The template is being saved in several test cases, not just in one. These test cases have been revisited and if needed revised.